packages/repository-json-schema/src/__tests__/integration/build-schema.integration.ts

```diff
@@ -988,14 +988,17 @@ describe('build-schema', () => {
         expect(originalSchema.title).to.equal('Product');
 
         const excludeIdSchema = getJsonSchema(Product, {exclude: ['id']});
-        expect(excludeIdSchema.properties).to.deepEqual({
-          name: {type: 'string'},
-          description: {type: 'string'},
+        expect(excludeIdSchema).to.deepEqual({
+          title: 'ProductExcluding_id_',
+          properties: {
+            name: {type: 'string'},
+            description: {type: 'string'},
+          },
+          not: {
+            anyOf: [{required: ['id']}],
```

Comment:
a question: why the generated excluding schema doesn't have `properties` field like:
```ts
not: {
  anyOf: [{properties: {id: {type: 'string'}}, required: ['id']}]
}
```
?